brave/mumble/service.py

```diff
@@ -162,7 +163,7 @@ def authenticate(self, name, pw, certificates, certhash, certstrong, current=Non
                 # Define the registration date if one has not been set.
                 Ticket.objects(character__name=name, registered=None).update(set__registered=datetime.utcnow())
         
-            for tag in ('member', 'blue', 'guest', 'mumble'):
+            for tag in ('admin', 'member', 'blue', 'guest', 'mumble'):
```

Comment:
Until we have proper documentation on group permissions and how they are constructed, or an app to build them, we should still support group names.